cattle/plugins/docker/compute.py

```diff
@@ -192,13 +192,54 @@ def _is_instance_active(self, instance, host):
 
     @staticmethod
     def _get_docker_client(host):
-        cluster_connection = ""
+        cluster_connection = None
+        tls_config = None
         try:
             cluster_connection = host['clusterConnection']
+            account_id = host['accountId']
+            ca_crt = host['caCrt']
+            client_crt = host['clientCrt']
+            client_key = host['clientKey']
+
```

Comment:
we should probably throw an error or log something, when some of the expected 3 params - ca_crt, client_crt, client_key - is missing, when some are present. As at the end we set the certificate, we expect those 3 params to be present.